sync/fromDisk/assessments.js

```diff
@@ -55,7 +55,11 @@ function getParamsForAssessment(assessmentInfoFile, questionIds) {
     number: assessment.number,
     title: assessment.title,
     multiple_instance: assessment.multipleInstance ? true : false,
-    shuffle_questions: assessment.shuffleQuestions ? true : false,
+    shuffle_questions:
```

Comment:
I'd encourage you to add a few tests for this behavior to https://github.com/PrairieLearn/PrairieLearn/blob/master/tests/sync/assessmentsSync.test.js:

* Check that `shuffleQuestions` defaults to `true` for Exam-type assessments
* Check that `shuffleQuestions` can be overridden to `false` for Exam-type assessments

Reply:
Added the suggested tests in cede76e